pending/smart_switch_pc_and_kies.xml

```diff
@@ -12,53 +12,79 @@
-    <action command="delete" search="glob" path="%ProgramData%\Samsung\Device Error Recovery\default-*.log"/>
+    <description>Delete the logs</description>
+    <action command="delete" search="walk.files" path="%ProgramData%\Samsung\Device Error Recovery\default-*.log"/>
   </option>
   <option id="device_profile_cache">
     <label>Device Profile Cache</label>
-    <description>Delete the Device Profile Cache</description>
+    <description>Delete the cache</description>
     <action command="delete" search="walk.all" path="%ProgramData%\Samsung\DeviceProfile\Cache\"/>
-    <action command="delete" search="glob" path="%ProgramData%\Samsung\DeviceProfile\Cache\"/>
   </option>
   <option id="device_roms">
     <label>Device ROMs</label>
     <description>Delete the downloaded Device ROMs</description>
-    <action command="delete" search="walk.all" path="%LOCALAPPDATA%\Temp\Binary\"/>
-    <action command="delete" search="glob" path="%LOCALAPPDATA%\Temp\Binary\"/>
+    <action command="delete" search="walk.all" path="%Temp%\Binary\"/>
+    <action command="delete" search="file" path="%Temp%\Binary\"/>
   </option>
-  <option id="user_software_data">
-    <label>User Software Data</label>
-    <description>Delete the User Software Data</description>
-    <action command="delete" search="walk.all" path="%APPDATA%\Samsung\"/>
-    <action command="delete" search="glob" path="%APPDATA%\Samsung\"/>
+  <option id="kies_user_software_data">
+    <label>Kies User Software Data</label>
+    <!-- Delete the old Kies User Software Data -->
+    <description>Delete the files</description>
+    <action command="delete" search="walk.all" path="%AppData%\Samsung\Kies\"/>
+    <action command="delete" search="file" path="%AppData%\Samsung\Kies\"/>
+    <action command="delete" search="walk.all" path="%AppData%\Samsung\Kies3.0\"/>
+    <action command="delete" search="file" path="%AppData%\Samsung\Kies3.0\"/>
   </option>
   <option id="selfmv_data">
     <label>SelfMV Data</label>
-    <description>Delete the SelfMV Data</description>
-    <action command="delete" search="walk.all" path="%USERPROFILE%\Documents\SelfMV\"/>
-    <action command="delete" search="glob" path="%USERPROFILE%\Documents\SelfMV\"/>
+    <description>Delete the files</description>
+    <action command="delete" search="walk.all" path="%UserProfile%\Documents\SelfMV\"/>
+    <action command="delete" search="file" path="%UserProfile%\Documents\SelfMV\"/>
+  </option>
+  <option id="temp">
+    <label>Temporary files</label>
+    <description>Delete the temporary files</description>
+    <action command="delete" search="walk.all" path="%Temp%\Binary\"/>
+    <action command="delete" search="file" path="%Temp%\Binary\"/>
+    <action command="delete" search="walk.all" path="%Temp%\Samsung\"/>
+    <action command="delete" search="file" path="%Temp%\Samsung\"/>
+    <action command="delete" search="glob" path="%Temp%\SmartSwitch_*\"/>
```

Comment:
On line 76 do you want to delete everything under `%Temp%\SmartSwitch_*\`? It seems you want `search="walk.all"`

Reply:
I think it should be "walk.all" and then "glob" to delete the directories, too?